pkg/sip/inbound.go

```diff
@@ -214,12 +238,32 @@ func (c *inboundCall) handleInvite(ctx context.Context, req *sip.Request, tx sip
 	defer c.close("other")
 	// Send initial request. In the best case scenario, we will immediately get a room name to join.
 	// Otherwise, we could even learn that this number is not allowed and reject the call, or ask for pin if required.
-	roomName, identity, wsUrl, token, requirePin, rejectInvite := c.s.dispatchRuleHandler(ctx, c.from.Address.User, c.to.Address.User, c.to.Address.Host, c.src, "", false)
-	if rejectInvite {
+	disp := c.s.handler.DispatchCall(ctx, &CallInfo{
+		FromUser:   c.from.Address.User,
+		ToUser:     c.to.Address.User,
+		ToHost:     c.to.Address.Host,
+		SrcAddress: c.src,
+		Pin:        "",
+		NoPin:      false,
+	})
+	switch disp.Result {
+	default:
+		logger.Errorw("Rejecting inbound call", fmt.Errorf("unexpected dispatch result: %v", disp.Result), "from", c.from.Address.User, "to", c.to.Address.User, "to-host", c.to.Address.Host, "src", c.src)
```

Comment:
Could this error ever be triggered by bad user data? If so, we usually don't log these at the Error level as we like alarming on Errors messages.

Reply:
No, I don't think so. It will only happen if we add some other status to the protocol.